.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Before opening an issue**
+
+Questions should go to the GEE mailing list (https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/gee-oss)
```

Comment:
Users don't actually have any way to access this group. I suggest pointing them to the https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/google-earth-enterprise group instead

Reply:
Changed it.